tests/v1/worker/test_spyre_input_batch.py

```diff
@@ -18,9 +18,8 @@
 MAX_NUM_PROMPT_TOKENS = 64
 
 
-def _remove_requests(
-        input_batch: InputBatch, batch_size: int,
-        reqs: list[CachedRequestState]) -> tuple[set[str], list[int]]:
+def _remove_requests(input_batch: InputBatch, batch_size: int,
+                     reqs: list[CachedRequestState]) -> set[str]:
```

Comment:
The function doc needs to be changed, it still says this function returns a `tuple`